src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -145,6 +154,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                                                                 : userEmailForCourse;
                     response.giverSection = userSectionForCourse;
                     responsesForQuestion.add(response);
+                    extractFeedbackResponseCommentsForResponseData(questionAttributes, questionIndx,
```

Comment:
Because we want to save comments only when there is a response.

Reply:
> Because we want to save comments only when there is a response.

But what if the response and comment are submitted together and response is not yet saved in the datastore.

Reply:
When will `response.responseMetaData.getValue().isEmpty()` be empty?

Reply:
responseMetaData is set to null to represent missing response.